reVX/plexos/rev_reeds_plexos.py

```diff
@@ -465,9 +571,25 @@ def _parse_plexos_nodes(plexos_nodes):
         plexos_nodes : pd.DataFrame
             Plexos node meta data including gid, lat/lon, voltage
         """
-        plexos_nodes = parse_table(plexos_nodes)
-        plexos_nodes = DataCleaner.pre_filter_plexos_meta(plexos_nodes)
+
+        if (isinstance(plexos_nodes, str)

```

Comment:
Why not add a sub if/else under isisnstance(plexos_nodes, str)?

Reply:
same effect isnt it? And adds "complexity" in the form of more levels. Bad incentives by pylint!!